apps/web/tests/activity-graph.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { test, expect } from '@playwright/test';
+
+import { getByTestId, initializeSession } from './utils.ts/browser';
+import { createNotifications } from './utils.ts/plugins';
+
+let session;
+
+test.beforeEach(async ({ context }) => {
+  session = await initializeSession(context);
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** This scope is always scoped to the file level right? Not for the global context of the tests running?‏

Reply:
I think yes, as every test is getting it's "session"... but we can improve it with fixtures that I mentioned here: https://github.com/novuhq/novu/pull/5313/files#r1564883613